### DISCUSSION TITLE
Is there a way to put the top on a full screen window?

### DISCUSSION THREAD

**Discussion by baigu7** [WindowTop/WindowTop-App#114] [General]

**Comment by gileli121** [comment]
@3456668888 

Usually, in games, there is an option to set **borderless fullscreen windowed mode.**
Read here about this option:
https://www.pcgamingwiki.com/wiki/Glossary:Borderless_fullscreen_windowed

Search how to do it in the game. I think that in most games there is such an option.
After that, you will able to put windows on top of the game.
